full/Dockerfile

```diff
@@ -1,5 +1,11 @@
 FROM ghcr.io/ledgerhq/ledger-app-builder/ledger-app-builder-lite:latest
 
+ARG RUST_STABLE_VERSION=1.72.0
+ARG RUST_NIGHTLY_VERSION=nightly-2022-12-02
```

Comment:
I can't remember why we pinned this nightly version. It feel fairly old already. If we know why, we should add a comment explaining it. Else, mabe we could use a newer version?

Reply:
At that time it was pretty recent. But yeah we should update it.